src/features/counter/Counter.module.css

```diff
@@ -0,0 +1,78 @@
+.row {
```

Comment:
Use relative dimensions like % and move the colors to a variable in the top of the file, @Kunal3737